include/arch/x86/arch/machine/fpu.h

```diff
@@ -114,7 +114,7 @@ static inline void disableFpu(void)
 #ifdef CONFIG_VTX
 static inline bool_t vcpuThreadUsingFPU(tcb_t *thread)
 {
-    return thread->tcbArch.tcbVCPU && &thread->tcbArch.tcbVCPU->fpuState == NODE_STATE(ksActiveFPUState);
```

Comment:
So this line is currently incorrect, and probably the whole reason why `ksActiveFPUState` wasn't a TCB to begin with.